arcade/examples/sprite_rotate_tank.py

```diff
@@ -0,0 +1,185 @@
+"""
+Sprite Rotation With A Tank.
+
+The problem often fis that vehicles or tower defense turrets can have parts that can rotate toward targets.
+These parts are usually represented with separate sprites drawn relative to attachment points on the main body.
+
+Because these sprites are usually asymmetrical, we have to rotate them around their attachment points on the main body,
```

Comment:
Try to limit line lengths in docstrings. [PEP-8 recommends 72 characters per line as the maximum for docstrings](https://peps.python.org/pep-0008/#maximum-line-length). You don't have to meet that exactly, but shorter line lengths than what you currently have are a good idea. You can split sentences across multiple lines safely as long as you aren't introducing a blank line right afterward.